CenterofExcellenceALMAccelerator/SolutionPackage/CanvasApps/cat_poweropsdevopsedition_7f3f4_DocumentUri_msapp_src/Src/DeploymentSettingsScreen.fx.yaml

```diff
@@ -3472,6 +3473,24 @@ DeploymentSettingsScreen As screen:
             Y: =204
             ZIndex: =43
 
+        chkUpgradeSolution As checkbox:
+            Default: =If(solutionupgradetag="upgrade",true,false)
```

Comment:
If refactored as above this would be Default: = solutionupgradetag

Reply:
Refractored the code by setting true/false instead of upgrade/update. However, true/false had to be set as text and used below condition to set the checkbox 'Default'.
If(solutionupgradetag="true",true,false)